Framework/Algorithms/src/CalculateTransmission.cpp

```diff
@@ -187,7 +186,7 @@ void CalculateTransmission::exec() {
     beamMonitorIndex = getIndexFromDetectorID(*sampleWS, beamMonitorID);
     logIfNotMonitor(sampleWS, directWS, beamMonitorIndex);
 
-    BOOST_FOREACH (size_t transmissionIndex, transmissionIndices)
+    for (auto transmissionIndex : transmissionIndices)
```

Comment:
This could also be done with [std::find](http://en.cppreference.com/w/cpp/algorithm/find).